packages/amazonq/src/lsp/client.ts

```diff
@@ -64,6 +64,7 @@ export async function startLanguageServer(
         '--stdio',
         '--pre-init-encryption',
         '--set-credentials-encryption-key',
```

Comment:
these are the flags for Flare. You need to add the flag for node binary itself.. If you cannot do it in argv, you can pass as environment variables